control-station/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,9 @@
+import "./style.css"
+import HX from "../../../public/Data/Images/HX Logo.png"
```

Comment:
Chore: assets in `public` cannot be imported from JavaScript ([Static Asset Handling, The `public` directory](https://vitejs.dev/guide/assets.html#the-public-directory)). Consider moving this file to `src/assets/` to take advantage of webpack's file loader.

Reply:
We might also want to consider using [path aliases](https://vitejs.dev/config/shared-options#resolve-alias) so we don't need to specify full relative paths.